File: java-play/app/controllers/Application.java
package controllers;

import play.*;
import play.mvc.*;
import play.data.Form;
import play.data.*;
import models.User;
import views.html.*;


public class Application extends Controller {
	
    static Form<User> userForm = Form.form(User.class);
    public static Result index() {
    	return ok(index.render(userForm));
    }
    
    public static Result submit() {
    	Form<User> filledForm = userForm.bindFromRequest();
    	User created = filledForm.get();
    	return ok(submit.render(created));
    }
}

File: practica_MVC - copia/app/controllers/Application.scala
package controllers

import play.api._
import play.api.mvc._
import play.api.data._
import play.api.data.Forms._
import models._
import views._

object Application extends Controller {


val itemForm = Form("item" -> nonEmptyText)

 
    def index = listItems
 
    def listItems() = Action{
   Ok(views.html.index(Item.list(), itemForm))
                          }

    def createItem() =Action { implicit request =>
   itemForm.bindFromRequest.fold(errors=> BadRequest(views.html.index(Item.list(), errors)),
      item => {Item.create(item)
      Redirect(routes.Application.listItems)
              }
                                )
                          }
   def deleteItem(id: Long) =Action{
   Item.delete(id)
   Redirect(routes.Application.listItems)
   }
 }